src/components/QuickOpenModal.js

```diff
@@ -121,33 +127,59 @@ export class QuickOpenModal extends Component<Props, State> {
     this.setState({ results });
   };
 
+  generalQuery = (query: string) => {
+    const results = formatGeneralQuery();
+    this.setState({ results });
+    return;
+  };
+
   updateResults = (query: string) => {
     if (this.isSymbolSearch()) {
       return this.searchSymbols(query);
     }
+    if (this.isGeneralQuery()) {
+      return this.generalQuery(query);
+    }
     return this.searchSources(query);
   };
 
+  updateQuery = (item: FormattedGeneralQuery) => {
+    const { setQuickOpenQuery } = this.props;
+    if (item.id == "id.123") {
```

Comment:
lol - perhaps @wldcordeiro has a better idea for the ids. I'm inclined to use a shortened name like gotoLine or searchFunctions

Reply:
I'm gonna agree the IDs should be stuff like `gotoLine`, `searchFunctions`, `searchVariables`, etc. Maybe just reuse the values from the `modifiers` map in the `quick-open.js` util file?